src/pages/community-partners.astro

```diff
@@ -48,13 +48,28 @@ const partners = [
 ---
 
 <Layout metaTitle="Community Partners | Dev Protocol">
-<div class="mt-28 lg:mt-20 max-w-full text-center">
-  <h2 class="mx-auto w-full font-display text-5xl m-8 dark:text-white">Community Partners</h2>
-  <h4 class="mx-auto w-full font-display text-xl m-4 p-8 dark:text-gray-100">Community Partners help us educate people about web3 and, become a part of their creator success</h4>
+<div class="flex flex-col md:gap-y-4 gap-y-8 mt-28 max-w-full text-center">
+  <h2 class="mx-auto w-full font-display text-5xl dark:text-white">Community Partners</h2>
+  <h4 class="mx-auto w-full font-display text-xl px-8 md:p-0 dark:text-gray-100">Community Partners help us educate people about web3 and, become a part of their creator success</h4>
+  <a href="#">
```

Comment:
The [issue](https://github.com/web3community/devprotocol.xyz/issues/187) didn't specify where the button should point to, so I leave it empty for now.